### PR TITLE
enable import sanity test

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: plugins/module_utils/foreman_helper.py [hunk 1]
```diff
@@ -233,6 +238,9 @@ def check_requirements(self):
         if not HAS_APYPIE:
             self.fail_json(msg='The apypie Python module is required', exception=APYPIE_IMP_ERR)
 
+        if not HAS_PYYAML:
+            self.fail_json(msg='The PyYAML Python module is required', exception=PYYAML_IMP_ERR)
```
Comment (sean797) [inline plugins/module_utils/foreman_helper.py]: Is it worth DRY'ing this? Something like? Perhaps even in module_utils

```
if FAILED_IMPORT:
    self.fail_json(msg='The {} Python module is required'.format(FAILED_IMPORT), exception=IMP_ERR)

Comment (evgeni) [reply plugins/module_utils/foreman_helper.py]: Dunno. There is #328 for when we decide to depend on Ansible 2.8, which is kinda DRYing it?

Comment (evgeni) [reply plugins/module_utils/foreman_helper.py]: One thing to think about: We *really* require `PyYAML` only in modules that call `parse_template(_from_file)`, so `foreman_job_template`, `foreman_provisioning_template` and `foreman_ptable`. Maybe this `fail` should be moved there?

Comment (sean797) [reply plugins/module_utils/foreman_helper.py]: Maybe, we just need to be sure to do the same if/when we create modules for more template types

Comment (evgeni) [reply plugins/module_utils/foreman_helper.py]: All (current) modules call `parse_template` before doing any *real* API interaction, so we could just do the `fail` inside `parse_template`?

Comment (sean797) [reply plugins/module_utils/foreman_helper.py]: I think thats a nice idea:1st_place_medal:

Comment (evgeni) [reply plugins/module_utils/foreman_helper.py]: I can't read. pushed.

**Pull request merged**